src/PostProcessRendering.cpp

```diff
@@ -0,0 +1,267 @@
+#include "PostProcessRendering.h"
+#include "Prelude.h"
+
+extern uint8_t r_image_font_6_11[2464];
+
+/*
+ * Context for the software renderer.
+ */
+struct BitmapData
+{
+    U32  w, h;
```

Comment:
Prefer using full words or very clear prefixes and abbreviations. It may be clear from context here what w and h are, but it's going to be used in unclear context (such as accessing it as r->w for example) and it just reduces readability. As Miroslav said, if this is code imported from somewhere external and that's why you didn't want to change it, include license/source in a comment.

Reply:
I prefer to derive meaning of variables from the function and structure being used. 'r' was suppose to mean renderer or render.
However, I understand this may cause confusion as not everyone thinks the same.
This structure has been updated, it has better names now (I think).